engine/env/spark/src/main/java/org/datacleaner/spark/Main.java

```diff
@@ -96,25 +97,50 @@ public static void main(String[] args) {
         }
     }
 
-    private static String getResultJobFilePath(final JavaSparkContext sparkContext,
-            final SparkJobContext sparkJobContext)  {
+    /**
+     * Gets the hdfs path of job's result
+     * 
+     * @return
+     */
+    protected static String getResultFilePath(final JavaSparkContext sparkContext, final SparkJobContext sparkJobContext) {
```

Comment:
Can we move these methods to a Util class instead of Main? Preferably the Main class is only for spark-submit and reading the command line arguments into some sensible object.